### ISSUE TITLE
Spring Boot with Reactor Netty Performance Issue right after application started

### ISSUE THREAD

**Issue by cjr333** [reactor/reactor-netty#1150]
I wrote a spring boot application and sample api. I found out the performance between 1st race(right after application started) and 2nd race(I think it is warmed up) was remarkably different. I guess it is caused some resources initialization - threads, direct byte buf, etc. But I can't find out exact cause and solution. I'm looking forward to your help and I want to know how many people had same experience.

[Specific Info] 
Spring boot: 2.2.4 
Reactor Netty: 0.9.4 
Sample api: just return Mono.just(Collections.singletonList(1)); 
Test condition: request 3000 TPS during 30 seconds 
Test Result: response time 
   - 1st race(right after application started): mean 574ms, p99 10004ms 
   - 2nd race: mean 5ms, p99 8ms

**Comment by violetagg** [comment]
@cjr333 Would you be able to give us a reproducible example (please use the latest releases)?

**Comment by cjr333** [comment]
Thank you for your immediate reply.
I simplified my app and push on my github repository. It is very simple and only have one api "/sample". ( https://github.com/cjr333/spring-boot.git )
First I tested with Spring boot 2.2.4 and get same result. And then I updated Spring boot to the latest release(2.3.1) and tested, but also get same result.
When I benchmark java apps performance, the overhead of class loading by class loader affect the result. So warm-up is needed. Does this case have same issue?? (I'm wondering)

**Comment by ericbottard** [comment]
@cjr333  Could you please elaborate how you measured response time (ie what tool and config was used), that would be great.

**Comment by cjr333** [comment]
@ericbottard 
I used gatling open source version for measurement.  (https://gatling.io/)
100 clients were used so each client made 30 requests per sec.

**Comment by douglashg** [comment]
How can I configure Spring 3 to use quic?

**Comment by violetagg** [comment]
@douglashg I don't think that in Spring Boot there is an integration with `reactor-netty-incubator-quic`
